src/main/java/au/org/ala/biocache/dao/SolrIndexDAOImpl.java

```diff
@@ -84,6 +80,19 @@
   @Inject
   private FieldMappingUtil fieldMappingUtil;
 
+  /*
+  * The csv header for updating solr
+  */
+  private final List<String> header = Arrays.asList(
+          "user_assertions"
+  );
+
+  /*
+  * The list of field that can hold multiple values
+  */
+  private final List<String> multiValueField = Arrays.asList(
```

Comment:
the SOLR index field should be `assertionUserId`. We've adopted camel case for all fields in the search index.

See the pipelines schema for reference


https://github.com/gbif/pipelines/blob/dev/livingatlas/solr/conf/managed-schema#L473